po/pt_BR.po

```diff
@@ -62,476 +65,483 @@ msgid ""
 
 #: data/io.github.lainsce.Notejot.appdata.xml.in:100
 msgid "New UI"
-msgstr ""
+msgstr "Nova interface"
 
 #: data/io.github.lainsce.Notejot.appdata.xml.in:101
 msgid "Using another widget to display note contents"
-msgstr ""
+msgstr "Usando outro componente para exibir conteúdo da nota"
 
 #: data/io.github.lainsce.Notejot.appdata.xml.in:102
 msgid "Keyboard shortcuts for formatting"
-msgstr ""
+msgstr "Atalhos de teclado para formatação"
 
 #: data/io.github.lainsce.Notejot.desktop.in:3
 msgid "Notejot"
-msgstr ""
+msgstr "Notejot"
 
 #: data/io.github.lainsce.Notejot.desktop.in:7
 msgid "text;plain;plaintext;notepad;notes;sticky;post-it;"
-msgstr ""
+msgstr "text;plain;plaintext;notepad;notes;sticky;post-it;texto;notas;"
 
 #: data/io.github.lainsce.Notejot.gschema.xml:5
 msgid "Migration Version"
-msgstr ""
+msgstr "Versão de Migração"
 
 #: data/io.github.lainsce.Notejot.gschema.xml:6
 msgid "The version of migration to use; Do not change."
-msgstr ""
+msgstr "A versão de migração a usar; Não mude."
 
 #: data/io.github.lainsce.Notejot.gschema.xml:10
 msgid "Last View Used"
-msgstr ""
+msgstr "Última visualização usada"
 
 #: data/io.github.lainsce.Notejot.gschema.xml:11
 msgid "The last view used in the UI"
-msgstr ""
+msgstr "A última visualização usada na interface"
 
 #: data/io.github.lainsce.Notejot.gschema.xml:15
 msgid "Notes Font Size"
-msgstr ""
+msgstr "Tam. da Fonte das Notas"
 
 #: data/io.github.lainsce.Notejot.gschema.xml:16
 msgid "The size of the font used in notes. (Small, Medium, Large)"
-msgstr ""
+msgstr "O tamanho da fotne usada nas notas (pequeno, médio, grande)"
 
 #: data/io.github.lainsce.Notejot.gschema.xml:20
 msgid "Saves whether the window is maximized or not"
-msgstr ""
+msgstr "Salva se a janela está maximizada ou não"
 
 #: data/io.github.lainsce.Notejot.gschema.xml:21
 msgid ""
 "Whether the window is maximized. Used to open window with user's preferred "
 "settings"
 msgstr ""
+"Se a janela está maximizada. Usada para abrir a jenale configurada de acordo "
+"com a preferência do usuário"
 
 #: data/io.github.lainsce.Notejot.gschema.xml:25
 msgid "Saves window last-used width"
-msgstr ""
+msgstr "Salva última largura usada para a janela"
 
 #: data/io.github.lainsce.Notejot.gschema.xml:26
 msgid "The user's last used window width before closing the app"
 msgstr ""
+"A última largura usada pelo usuário para a janela antes de fechar o programa"
 
 #: data/io.github.lainsce.Notejot.gschema.xml:30
 msgid "Saves window last-used height"
-msgstr ""
+msgstr "Salva última altura usada para a janela"
 
 #: data/io.github.lainsce.Notejot.gschema.xml:31
 msgid "The user's last used window height before closing the app"
 msgstr ""
+"A última altura usada pelo usuário para a janela antes de fechar o programa"
 
 #: src/MainWindow.vala:261
 msgid "Jot your ideas."
-msgstr ""
+msgstr "Anote suas idéias."
 
 #: src/MainWindow.vala:267
 msgid "translator-credits"
-msgstr ""
+msgstr "translator-credits"
```

Comment:
This is to add your name and email as in line 7. It will appear in the About dialog.
```suggestion
msgstr "Clovis Fabricio <github.nosklo@0sg.net>"
```